requirements/key-revocation.md

```diff
@@ -1,6 +1,6 @@
 # Key Revocation
 
-One of the goals of Notary v2 is to build in solutions for key revocation that are easy to use and ensure that users will always use non-compromised keys.
+One of the goals of Notary Project is to build in solutions for key revocation that are easy to use and ensure that users will always use non-compromised keys.
```

Comment:
One of the goals of Notary Project is to build in solutions for key revocation that are easy to use ~~and ensure that users will always use non-compromised key.~~

Reply:
Perhaps not part of this change, but we need a separate PR to overhaul this markdown file. For almost all of the references to the word "key' on this page should be replaced with "certificate", or we should state that upfront in the doc that Key refers to certificates

cc: @priteshbandi , @gokarnm

Reply:
@iamsamirzon created an issue for tracking your comment https://github.com/notaryproject/notaryproject/issues/269, and this PR only addressed naming issue.